python/mxnet/gluon/data/dataloader.py

```diff
@@ -299,7 +278,7 @@ def shutdown(self):
             self._shutdown = True
 
 
-class DataLoader(object):
+class DataLoaderV1(object):
```

Comment:
If some specific implementations were relying on the old queue based methods, I think leaving the older version is still preferable for some user, what do you think?

Reply:
I don't have a strong opinion about it. If you deem the reliance on queue as reliance on an undocumented implementation detail of V1, then it may be better to remove from Gluon and copy the V1 code over to the specific implementation? It may be reduce the number of APIs to maintain in the future. But I'm also OK with keeping it here if you prefer